### DISCUSSION TITLE
How to Revenue Tracking on self-hosting

### DISCUSSION THREAD

**Discussion by Bossert-WebConcept** [plausible/analytics#3611] [Self-Hosted Support]
Hey there,
how is it possible to set up Custom Events with Revenue Tracking?
I don't have the button "Revenue" like explained in the docs

**Comment by ruslandoga** [comment]
👋 @bwconcept 

AFAIK the latest `v2` container image doesn't include the revenue tracking functionality.

**Reply by Giovxiii** [reply]
Then Plausible should write it on their doc page to make us save time!
https://plausible.io/docs/ecommerce-revenue-tracking

No plans to add it on self-hosting in the future ?
